packages/libs/hd-wallet/src/chainweaver/vendor/kadena-crypto.js

```diff
@@ -8360,8 +8362,8 @@ if (debugUtil && debugUtil.debuglog) {
 }
 /*</replacement>*/
 
-var BufferList = require('./internal/streams/BufferList.js');
-var destroyImpl = require('./internal/streams/destroy.js');
+var BufferList = require('./internal/streams/BufferList');
```

Comment:
not sure if it matters, but potentially the removal of `.js` makes this extraction `esm` incompatible?

Reply:
this bundle is created with browserify, I have copied it from the old lib.
it only work if I remove `.js` actually I didnt add them manually my VSCode did it when I move the file to the different path. 
I will check `esm` compatibility